pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -218,12 +218,36 @@ func (r *ReconcileNode) createOrUpdateNodeDaemons(node corev1.Node, tolerations
 			logger.Debugf("crash collector successfully reconciled for node %q. operation: %q", node.Name, op)
 		}
 	}
+	if cephCluster.Spec.Monitoring.Enabled {
+		op, err := r.createOrUpdateCephExporter(node, tolerations, cephCluster, cephVersion)
+		if err != nil {
+			if op == "unchanged" {
+				logger.Debugf("ceph exporter unchanged on node %q", node.Name)
+			} else {
+				return errors.Wrapf(err, "ceph exporter reconcile failed on op %q", op)
+			}
+		} else {
+			logger.Debugf("ceph exporter successfully reconciled for node %q. operation: %q", node.Name, op)
+			// create the metrics service
+			service, err := MakeMetricsService(cephCluster, "ceph-exporter-http-metrics", r.scheme)
+			if err != nil {
+				return err
+			}
+			if _, err := k8sutil.CreateOrUpdateService(r.opManagerContext, r.context.Clientset, cephCluster.Namespace, service); err != nil {
+				return errors.Wrap(err, "failed to create ceph-exporter metrics service")
+			}
+
+			if err := EnableServiceMonitor(cephCluster, r.scheme, r.opManagerContext); err != nil {
+				return errors.Wrap(err, "failed to enable service monitor")
+			}
```

Comment:
How about adding a debug log entry here that the service monitor was reconciled?